### PR TITLE
Decrease number of requests in tests

### PR DESCRIPTION
**Pull request by ihoru**
Use setUpClass instead of __init__ in BaseTest.

In my case speed change was: 324s->240s (25%)

### CODE REVIEW

**Comment by Eldinnie** [comment]
That would make for a descent increase indeed. Maybe you can take a look at `test_video` etc. It uses a new class (`bots.py`) to get a bot. This is to later implement using multiple bot's in one easy place.
I would appreciate it if you could

1. rewrite `BaseTest` to use `get_bot()`
2. remove `get_bot()` from `test_video`, `test_audio`, etc (all the media tests)

**Comment by ihoru** [comment]
@Eldinnie , good notice! Done.

File: tests/test_audio.py [hunk 1]
```diff
@@ -25,25 +25,22 @@
...
 from tests.base import BaseTest, timeout
-from tests.bots import get_bot
 
 
 class AudioTest(BaseTest, unittest.TestCase):
     """This object represents Tests for Telegram Audio."""
 
     @classmethod
     def setUpClass(cls):
+        super().setUpClass()
```
Comment (Eldinnie) [inline tests/test_audio.py]: To support python 2 (which we do) you need to use `super(TestAudio, self).setUpClass(cls)` (see Travis' logs)
Same for all others

Comment (ihoru) [inline tests/test_audio.py]: Oh, yeah... I did this mistake previously... Didn't learn that lesson :) Now I did ;)
Fixed.

**Comment by Eldinnie** [comment]
Looks good to me

**Comment by Murod1997** [comment]
привет

**Pull request merged**